extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -999,7 +999,7 @@ export class AmpSlideScroll extends BaseCarousel {
 
   /**
    * @param {string} eventType
-   * @param {!JsonObject} vars A map of vars and their values.
+   * @param {!Object} vars A map of vars and their values.
```

Comment:
Why does this change?

Reply:
JsonObject doesn't have a TS equivalent afaik. cc @jridgewell

Reply:
JSON object is equivalent to an `any`. But, we can [finally type](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#more-recursive-type-aliases) this, if we want:

```ts
type Json =
  | string
  | number
  | boolean
  | null
  | JsonObject
  | JsonArray;
type JsonObject = { [property: string]: Json };
type JsonArray = Json[];
```

Reply:
^^^ @samouri can you either add this, or add an alias for `JsonObject=any` in a shame file so we can leave the annotations here and narrow them later?